### ISSUE TITLE
About the experimental settings in the paper "HIFI-CODEC: GROUP-RESIDUAL VECTOR QUANTIZATION FOR HIGH FIDELITY AUDIO CODEC"

### ISSUE THREAD

**Issue by Shengqiang-Li** [yangdongchao/AcademiCodec#51]
I want to reproduce the results in this paper "HIFI-CODEC: GROUP-RESIDUAL VECTOR QUANTIZATION FOR HIGH FIDELITY AUDIO CODEC". However, the description is so confused. The paper just said that the training data comes from public datasets the public datasets, such as LibriTTS, VCTK, AISHELL. I have some questions：
1. Which dataset is included in the train set and test set?
2. How many utterances are there in the train set and test set respectively? 
It would be nice if you could provide a bit more information.

**Comment by yangdongchao** [comment]
> I want to reproduce the results in this paper "HIFI-CODEC: GROUP-RESIDUAL VECTOR QUANTIZATION FOR HIGH FIDELITY AUDIO CODEC". However, the description is so confused. The paper just said that the training data comes from public datasets the public datasets, such as LibriTTS, VCTK, AISHELL. I have some questions：
> 
> 1. Which dataset is included in the train set and test set?
> 2. How many utterances are there in the train set and test set respectively?
>    It would be nice if you could provide a bit more information.

We donot split test set from the public datasets. The test set from our intern dataset, includes 1000 high-quality utterances.
